DulcificumJS/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@ultimaker/dulcificumjs",
+  "version": "0.1.0-alpha.4",
+  "description": "DulcificumJS is a TS component to run Dulcificum in a browser",
+  "main": "dist/index.js",
+  "scripts": {
+    "install_dulcificum": "conan install dulcificum/latest@ultimaker/np_76 -s build_type=Release --build=missing --update -c tools.build:skip_test=True -pr:h cura_wasm.jinja -if src/Dulcificum && rm -f src/Dulcificum/*conan*",
```

This should point back to the main branch before merging
```suggestion
    "install_dulcificum": "conan install dulcificum/latest@ultimaker/testing -s build_type=Release --build=missing --update -c tools.build:skip_test=True -pr:h cura_wasm.jinja -if src/Dulcificum && rm -f src/Dulcificum/*conan*",
```